python/docs/source/migration_guide/pyspark_upgrade.rst

```diff
@@ -22,6 +22,8 @@ Upgrading PySpark
 Upgrading from PySpark 3.5 to 4.0
 ---------------------------------
 
+* In Spark 4.0, the minimum supported version for Pandas has been raised from 1.0.5 to 1.4.4 in PySpark.
```

Comment:
I move the message for `pandas` and `numpy` to the beginning, since I think it is more important, otherwise they will be surrounded by the messages of Pandas APIs.

Reply:
cc @itholic @dongjoon-hyun